client/js/main.js

```diff
@@ -0,0 +1,120 @@
+document.getElementById("velocity-time").width = document.getElementById("chart1").offsetWidth;
+document.getElementById("velocity-time").height = document.getElementById("chart1").offsetHeight;
+
+new Chart(document.getElementById("velocity-time"), {
+                type:'scatter',
+                data:{
+                    label:'Scatter Dataset',
+                    datasets:[{
+                    data:[{x:0, y:0},{x:5, y:2},{x:10, y:22},{x:15, y:40},{x:20, y:56},{x:25, y:70},{x:30, y:82},{x:35, y:92},{x:40, y:100}],
+                    backgroundColor:['rgba(165, 105, 189, 0.6)']
+                    }]
+                    },
+
+                options:{
```

Comment:
I would recommend creating a variable called `velocityTimeChartOptions` and storing what is here instead. (Do for the other charts too)